templates/kratos.yaml.j2

```diff
@@ -27,17 +27,6 @@ selfservice:
         login:
             ui_url: {{ login_ui_url }}
         {%- endif %}
-        {%- if registration_ui_url %}
-        registration:
-            enabled: True
-            ui_url: {{ registration_ui_url }}
-            {%- if oidc_providers %}
-            after:
-                oidc:
-                    hooks:
-                    - hook: session
```

Comment:
When user logs in for the first time, they are registered. If the session hook is not enabled, they will not be logged in (see https://www.ory.sh/docs/kratos/hooks/configure-hooks#available-actions). 

We can't remove this